src/main/java/de/fraunhofer/aisec/cpg/graph/declarations/FunctionDeclaration.java

```diff
@@ -247,6 +247,10 @@ public void addParameter(ParamVariableDeclaration paramVariableDeclaration) {
     this.parameters.add(propertyEdge);
   }
 
+  public void addParameterList(List<ParamVariableDeclaration> params) {
```

Comment:
This should not be done directly but rather through the scope manager's `addDeclaration`. This will automagically add it to the correct declaration "container"